src/commands/import.ts

```diff
@@ -433,6 +433,30 @@ const getProjectStatusField = async ({
   return response.node.field;
 };
 
+const updateProjectStatusField = async ({ octokit, statusFieldId, options }: { octokit: Octokit; statusFieldId: string; options: SelectOption[] }): Promise<Array<{ id: string; name: string }>> => {
```

Comment:
We usually describe arrays in TypeScript like this, so let's stick to the convention:

```suggestion
const updateProjectStatusField = async ({ octokit, statusFieldId, options }: { octokit: Octokit; statusFieldId: string; options: SelectOption[] }): Promise<{ id: string; name: string }[]> => {
```